GetColorFromPairNumber.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include "declare.h"
+ColorPair GetColorFromPairNumber(int pairNumber) 
+{
+const char* MinorColorNames[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
```

Comment:
minorColorNames is redefined multiple times.  this is a good candidate to be part of a single definition file for ex. a .h file.